### PR TITLE
feat(route): add AIEA seminars

### LINKED ISSUE #11288: AIEA upcoming seminars
### 类型

学习

### 网站地址

http://www.aiea.org/0504

### 网站描述

亚洲创新创业协会

### 需要生成什么内容？

需要生成近期的学术会议
谢谢

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by zxx457**
## 该 PR 相关 Issue / Involved issue
Close #11288

## 完整路由地址 / Example for the proposed route(s)

```routes
/aiea/seminars/past
/aiea/seminars/upcoming
/aiea/seminars/both
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/aiea/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const buildData = require('@/utils/common-config');
+
+module.exports = async (ctx) => {
+    const link = 'http://www.aiea.org/0504';
```
Comment (TonyRL) [inline lib/v2/aiea/index.js:4]: Please use HTTPS

Comment (zxx-457) [reply lib/v2/aiea/index.js:4]: The website does not use tls, should still change to https?

File: lib/v2/aiea/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    'seminars/:period': ['zxx-457'],
```
Comment (TonyRL) [inline lib/v2/aiea/maintainer.js:2]:
```suggestion
    '/seminars/:period': ['zxx-457'],
```

**Review by TonyRL — COMMENTED** [review]
Please add docs

File: lib/v2/aiea/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'aiea.org': {
+        _name: 'AIEA',
+        www: [
+            {
+                title: 'AIEA Seminar Series',
+                docs: 'https://docs.rsshub.app/study.html#AIEA-Seminar-Series',
+                source: '/504',
```
Comment (TonyRL) [inline lib/v2/aiea/radar.js:8]:
```suggestion
                source: ['/0504', '/']
```

**Pull request merged**